service-front/module/Application/src/Controller/General/FeedbackController.php

```diff
@@ -58,6 +65,14 @@ public function indexAction()
         $request = $this->convertRequest();
 
         if ($request->isPost()) {
+            $formGeneratedTime = $container->form_generated_time ?? 0;
+            unset($container->form_generated_time);
+
+            if ($this->dateService->getNow()->getTimestamp() - $formGeneratedTime < self::MIN_SUBMISSION_TIME_SECONDS) {
+                $this->getLogger()->warning('Feedback form submitted too quickly, possible bot submission');
+                return $this->redirect()->toRoute('feedback-thanks');
```

Comment:
I think this should throw an error rather than silently accept it, so that we don't lose data from people operating quickly (like someone just selecting the satisfaction rating)